src/main/java/org/protege/editor/owl/client/LocalHttpClient.java

```diff
@@ -687,13 +676,8 @@ public void deleteRole(RoleId roleId) throws AuthorizationException, ClientReque
 	@Override
 	public void updateRole(RoleId roleId, Role updatedRole)
 			throws AuthorizationException, ClientRequestException {
-		try {
-			manager.setRole(roleId, updatedRole);
-			putConfig();
-		} catch (UnknownRoleIdException e) {
-			logger.error(e.getMessage());
-			throw new ClientRequestException("Client failed to update role (see error log for details)", e);
-		}
+		config = new ConfigurationBuilder(config).setRole(roleId, updatedRole).createServerConfiguration();
```

Comment:
there are a few calls like this, where .eg. the `setRole` could fail, but it [swallows](https://github.com/protegeproject/metaproject/pull/5/files#diff-74b71838b24633e2fbe12a70b1302886R449) the exception, though arguably I can't see how the swallowed exception in `setRole` ever occurs, as the remove is just deleting from a hash.